wolfcrypt/src/aes.c

```diff
@@ -9523,6 +9537,9 @@ int wc_AesGcmEncryptFinal(Aes* aes, byte* authTag, word32 authTagSz)
  *
  * Must call wc_AesInit() before calling this function.
  *
+ * Call wc_AesGcmSetExtIV() before calling this function to use FIPS external IV
```

Comment:
Is `wc_AesGcmSetExtIV()` only used for FIPS (i.e. without long IVs), or is this API expected to work in non-FIPS scenarios (i.e. with long IVs), as a stand-in for `AesGcmSetIV()`?

Reply:
If you look at the implementation of wc_AesGcmSetExtIV() you will see that it copies the IV into aes->reg.
It then sets the invokeCtr as required by SP800-38D which is a NIST standard for FIPS.
This API was added for FIPS.

Reply:
I see!  Thank-you for the clarification. .:)